### PR TITLE
34 pybalt delete hardcode

### PR DESCRIPTION
**Pull request by pybalt**

### CODE REVIEW

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -7,14 +7,15 @@ type PropsAccordion = {
...
 }
 
-const Accordion = ({ data: { title, description, isOpen = false } }: PropsAccordion) => {
+const Accordion = ({ data: { title, description, isOpen = false }, key }: PropsAccordion) => {
   const [open, setOpen] = useState(isOpen);
   const descriptionRef = useRef(null);
 
   return (
-    <div className={styles.accordion}>
+    <div className={styles.accordion} key={key}>
```
Comment (bairon-cano) [inline src/components/Accordion/Accordion.tsx:18]: It is not necessary to receive the key as a prop, according to the react documentation, it is enough to put the key at component render time. I leave you the documentation, the first is the old documentation and the second is the latest.
[https://es.legacy.reactjs.org/docs/lists-and-keys.html](url)
[https://es.react.dev/learn/rendering-lists#recap](url)

File: src/pages/get-help/index.tsx [hunk 2]
```diff
@@ -7,8 +7,9 @@ const GetHelp = () => {
 
   return (
     <div className="container my-5">
-      <Accordion data={data[0]} />
-      <Accordion data={data[1]} />
+      {data.map((item, index) => (
+        <Accordion key={index} data={item} />
+      ))}
```
Comment (bairon-cano) [inline src/pages/get-help/index.tsx:12]: Putting the key like you did here is fine. Another thing that is not recommended is to use the index as a key, the reason is in the documentation, I think we could use the title instead.

**Review by bairon-cano — CHANGES_REQUESTED** [review]

File: src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (pybalt) [reply src/components/Accordion/Accordion.tsx]: Thanks!

File: src/pages/get-help/index.tsx (hunk 2 above)
Comment (pybalt) [reply src/pages/get-help/index.tsx:12]: I think I fixed it. Can you read it?

Comment (bairon-cano) [reply src/pages/get-help/index.tsx:12]: Everything is alright. I'll approve your changes.

**Review by bairon-cano — APPROVED** [review]
Well done!

**Pull request merged**